html-forms/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <style>
+      ul {
+        padding: 0;
+      }
+      li {
+        list-style-type: none;
+      }
+    </style>
+  </head>
+  <body>
+    <form action="" method="post">
+      <fieldset>
+        <h1>Coding Form!</h1>
+        <ul>
+          <li>
+            <span style="display: block"><label for="mail">Email:</label></span>
```

Comment:
If you want to utilize a block level element, why not use `p` instead of `span`?